codexctl/sync.py

```
@@ -216,0 +1,218 @@
+        self.logger = logger
+
+        if self.logger is None:
+            self.logger = logging
+
+        self.BASE = BASE
+        self.NAME_ATTRIBUTE = "VissibleName"
+        self.ID_ATTRIBUTE = "ID"
+
+        self.logger.debug(f"Base is: {BASE}")
+
+    def __POST(self, endpoint, data={}, fileUpload=False):
```

Ruff (0.8.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)